audformat/core/attachment.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+import audeer
+
+from audformat.core.common import HeaderBase
+
+
+class Attachment(HeaderBase):
+    r"""Database attachment.
+
+    Adds a file as attachment to a database.
+
+    Args:
+        path: relative path to file
+        description: attachment description
+        meta: additional meta fields
+
+    Raises:
+        ValueError: if ``path`` is absolute
+            or contains ``\``, ``..`` or ``.``
+
+    Example:
+        >>> Attachment('file.txt', description='Attached file')
+        {description: Attached file, path: file.txt}
+
+    """
+    def __init__(
+            self,
+            path: str,
+            *,
+            description: str = None,
+            meta: dict = None,
+    ):
+        super().__init__(description=description, meta=meta)
+
+        if (
+                os.path.isabs(path)
+                or '\\' in path
+                or path.startswith('./')
+                or '/./' in path
+                or path.startswith('../')
+                or '/../' in path
```

Comment:
I will move this to a utility function as we also use it inside `Database.is_portable()`

Reply:
done.